build_tools/docker/context/install_vulkan.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# Copyright 2023 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+set -e
```

Comment:
nit: Perform more checks and pipe the program errors

```suggestion
set -euo pipefail
```

Reply:
Will fix